### ISSUE TITLE
Test run - display of test case is now less readable

### ISSUE THREAD

**Issue by Prome88** [kiwitcms/Kiwi#764]
With the new desing of test case (where data from all four fields is stashed into one field) the display of test case on test run is now less readable than before.

6.4:
![image](https://user-images.githubusercontent.com/25688732/52210082-3ca9cb00-2886-11e9-86c8-85f92a629a96.png)

6.5:
![image](https://user-images.githubusercontent.com/25688732/52210233-b3df5f00-2886-11e9-926d-9e6a108de6f5.png)

If you don't blurr the text of scenarios (which also uses some bold text) it looks even worse.

Are there plans to update this page to patternfly and make it more readable? This is a stopper for upgrade to 6.5 for teams that use KiwiTCMS for manual test scenarios.

**Comment by atodorov** [comment]
Not being readable may be bad, but definitely not a show stopper. From what I can see font size in the new UI is bigger so the text should be easier to read.

What exactly do you mean by "not readable" and how do you imagine it should look like ?

The design side can be improved however internals are not going to change. A 4 text field variant of test case doesn't work for teams which use Given-When-Then and don't care about detailing individual actions. The 4 field design is also not going to work for anyone who uses a free-form text format and not the setup/actions/expected result/breakdown structure.

**Comment by Prome88** [comment]
Quite few  other commercial TCM tools which team members are accustomed to, use the four separate fields format for easier reading comprehansion of the test case instructions.


One would atleast expect one empty line or some kind of horizontal spacer to distinct between all the four stages of test case.
![image](https://user-images.githubusercontent.com/25688732/52211893-716c5100-288b-11e9-8bbb-206fb8787f81.png)

**Comment by atodorov** [comment]
We can easily adjust the spacing. Wouldn't work if you had already migrated but if you wait for the next update it will be better.

**Comment by asankov** [comment]
When we have migrated from 4 fields to 1, the migration merged the 4 fields using the following format:
```
**Setup:**
{{ test_case_text.setup }}

**Actions:**
{{ test_case_text.action }}

**Expected result:**
{{ test_case_text.effect }}

**Breakdown:**
{{ test_case_text.breakdown }}
```
(notice the empty lines). You can see that these lines are present if you open the Edit TestCase page (https://demo.kiwitcms.org/case/3658/edit/) and they are also nicely visible in the View TestCase page (https://demo.kiwitcms.org/case/3658/?from_plan=1468). However, these empty lines are not visible in the View TestRun page (https://demo.kiwitcms.org/runs/766/). 

Both the TestRun page and TestCase page use the function `markdown2html` to visualize the field. This function, when called with this text as an arguments returns the following HTML:
```html
<p>
  <strong>Setup:</strong><br/>
  {{ test_case_text.setup }}
</p>
<p>
  <strong>Actions:</strong><br/>
  {{ test_case_text.action }}
</p>
<p>
  <strong>Expected result:</strong><br/>
  {{ test_case_text.effect }}
</p>
  <p><strong>Breakdown:</strong><br/>
  {{ test_case_text.breakdown }}
</p>
```
This HTML is properly displayed in View Test Case page, because of the Patternfly classes that we use there, but not in View Test Run page, because there are no Patternfly classes there, to take care of the beautiful visualization of things.

TL;DR: This is going to be fixed out of the box, when we migrate View TestRun page to Parternfly

**Comment by atodorov** [comment]
Found the root-cause:
`.list td p{line-height:16px; margin:0px;}`

@asankov remove margin from this style definition and that will fix it.
